document available host preflights types (#656)
replicatedhq/kurl.sh: src/markdown-pages/create-installer/host-preflights/disk-usage.md

---
path: "/docs/create-installer/host-preflights/disk-usage"
date: "2022-01-13"
weight: 28
linktitle: "Disk Usage"
title: "Disk Usage"
---
 

> The disk usage host preflight check can be used to detect and validate the usage of a disk on which a particular directory lives.

Reviewer: ```suggestion
You can use the disk usage host preflight check to detect and validate the usage of a disk for particular directories.
```
